test/test-fwknop.pl

```diff
@@ -5910,7 +5925,15 @@ ()
         } else {
             ### pcap file mode, nothing to do
         }
-
+        my $quickcount = 0; ### Adds up to 5 seconds of time
+        while (&file_find_num_matches(qr/SPA\sData/, $NO_APPEND_RESULTS, $server_cmd_tmp) == 0 ) {
+            precise_sleep(.5);
+            print ".";
```

Comment:
Quick note - I'm assuming this print statement was used for debugging and not needed for the production code?